Makefile

```diff
@@ -1,10 +1,18 @@
 DOCKER_IMAGE ?= lev-report
+DOCKER_DB_IMAGE ?= lev-report-mock-db
 
-.PHONY: all clean deps docker docker-clean node-deps test unit-test
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
+
+compose_project_name = $(current_dir)
+compose_network_regexp != echo "$$(echo '$(compose_project_name)' | sed -E 's/([-_])+/[-_]*/g')_default"
+probe_network = docker network ls | grep -o '$(compose_network_regexp)'
+
+.PHONY: all clean deps docker docker-clean docker-test node-deps test unit-test
 
 all: deps test docker
 
-clean:
+clean: docker-compose-clean
```

Comment:
I wonder if this is better off in `distclean`?

Reply:
What do we do elsewhere?

Reply:
I think it makes more sense in `distclean`  looking at what that does already...